### PR TITLE
Reimplement the countdown

### PR DESCRIPTION
**Pull request by epsy**

### CODE REVIEW

File: pybay/templates/frontend/base.html [hunk 1]
```diff
@@ -175,7 +175,6 @@ <h1 class="text-center" style="color: black;">Proudly Sponsored By</h1>
         <script src="{% static 'new/js/spectragram.min.js' %}"></script>
         <script src="{% static 'new/js/flexslider.min.js'%}"></script>
         <script src="{% static 'new/js/jquery.plugin.min.js' %}"></script>
-        <script src="{% static 'new/js/jquery.countdown.min.js' %}"></script>
```
Comment (PirosB3) [inline pybay/templates/frontend/base.html:178]: remove file too

File: pybay/templates/frontend/index.html [hunk 2]
```diff
@@ -344,3 +344,13 @@ <h1 class="large-h1" style="color: black;">Subscribe to stay informed</h1>
     </section>
     <!--Subsribe to stay Tune Ends -->
 {% endblock content %}
+
+{% block scripts %}
+{{ block.super }}
+{% if countdown %}
+<script type="text/javascript" src="{% static 'new/js/countdown.js' %}"></script>
+<script type="text/javascript">
+countdown($('#countdown').get(0));
```
Comment (PirosB3) [inline pybay/templates/frontend/index.html]: getElementById

Comment (epsy) [reply pybay/templates/frontend/index.html]: Done.

File: pybay/templates/frontend/base.html (hunk 1 above)
Comment (epsy) [reply pybay/templates/frontend/base.html:178]: Done

**Review by PirosB3 — APPROVED** [review]

**Pull request merged**